### PR TITLE
[bitnami/postgresql] Add arbitrary securityContext configuration to improve security

### PR DESCRIPTION
**Pull request by djcass44**
**Description of the change**

Adds options to set arbitrary pod and container security context values.

**Benefits**

Improves the applications ability to run in strict security environments where certain privileges must be dropped or configured to a certain specification but aren't done so automatically (e.g. OPA GateKeeper in Kubernetes)

**Possible drawbacks**

`securityContext.runAsUser` was moved to `containerSecurityContext.runAsUser` as the original was only ever used in the container security context, not the pod one. This may be confusing when upgrading, however it maintains the original behaviour of the chart.

**Applicable issues**

  - references #3669

**Additional information**

- Based off #3615

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/postgresql/values-production.yaml [hunk 1]
```diff
@@ -82,9 +82,15 @@ volumePermissions:
 ## Pod Security Context
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 ##
-securityContext:
+podSecurityContext:
```
Comment (andresbono) [inline bitnami/postgresql/values-production.yaml:85]: This should still be `securityContext`, right?

**Review by andresbono — COMMENTED** [review]
Just a minor comment, the rest looks good to me!

File: bitnami/postgresql/values-production.yaml (hunk 1 above)
Comment (djcass44) [reply bitnami/postgresql/values-production.yaml]: Yes it should! I've pushed a new commit which fixes it

**Review by andresbono — APPROVED** [review]
LGTM!

**Comment by andresbono** [comment]
Thank for your contribution, @djcass44! :tada:

**Pull request merged**